example/sinatra/webapp.rb

```diff
@@ -0,0 +1,77 @@
+require 'veritrans'
+require 'json'
+require 'sinatra'
+
+Veritrans.setup do
+  config.load_yml File.dirname(__FILE__) + "/veritrans.yml#development"
```

Comment:
Why are this demo using `config.load_yml File.dirname(__FILE__) + "/veritrans.yml#development"`, while the lib readme doesn't even mention this method of setup via YML file? you're confusing merchant by being inconsistent.